Test/Tests/Network/SingleHostWithUtils.Tests.ps1

```diff
@@ -38,8 +38,14 @@ Describe "Single compute node protocol tests with utils" {
         Write-Log $("Setting a connection between " + $Container1NetInfo.MACAddress + `
         " and " + $Container2NetInfo.MACAddress + "...")
 
-        Invoke-Command -Session $Session -ScriptBlock {
+        Invoke-CommandWithFunctions `
```

Comment:
Please move the dlls check to separate function (`Test-IfUtilsCanLoadDlls`?) and check `vif --help` there